e2e/scenes/various-suite/frontend-sandbox-datasource.spec.ts

```diff
@@ -123,6 +138,10 @@ describe.skip('Datasource sandbox', () => {
         e2e.pages.Explore.visit();
         e2e.components.DataSourcePicker.container().should('be.visible').click();
         cy.contains(DATASOURCE_TYPED_NAME).scrollIntoView().should('be.visible').click();
+        // make sure the datasource was correctly selected and rendered
+        cy.get(`span[data-testid="${selectors.components.Breadcrumbs.breadcrumb(DATASOURCE_TYPED_NAME)}"]`).should(
+          'be.visible'
+        );
```

Comment:
you can simplify this further i think
```suggestion
        // make sure the datasource was correctly selected and rendered
        e2e.components.Breadcrumbs.breadcrumb(DATASOURCE_TYPED_NAME).should(
          'be.visible'
        );
```